meta/main.yml

```
@@ -1,10 +1,16 @@
 ---
 galaxy_info:
-  author: First Last
-  description: Skeleton Ansible role
+  author: Shane Frasier
+  description: >
```

Do we want this to have a trailing newline?

```suggestion
  description: >-
```